### ISSUE TITLE
View number of times Offer is redeemed

### ISSUE THREAD

**Issue by samgooch** [Merchello/Merchello#2062]
This may be a question or an enhancement. I can see there is a database table for exactly this purpose...

Is there a way to view how many times an offer has been redeemed? And over a configurable period in time?

**Comment by YodasMyDad** [comment]
There is actually a separate table for this, and it does record the amount of times redeemed. But it's not pulled through in that view.

**Comment by samgooch** [comment]
Yeah I saw that, it would be good if this report could be added to the Reports section. Or the Reports section is changed so that it is easily extendable.
